app/javascript/local/dropzone-cfg.js

```diff
@@ -12,8 +35,23 @@ DropzoneCfg.prototype.init = function () {
 
   if (!this.$dropzoneContainer || !this.$feedbackContainer) { return }
 
+  // As dropzone and the fallback components share the parent form component, dropzone specific styling is added here
+  this.$dropzoneContainer.classList.add("dropzone")
+
+  this.$chooseFilesButton = document.querySelector('button#choose_files_button')
+
   // Display of choose files button is conditional on whether JS is enabled
-  document.getElementById('choose_files_button').classList.remove("govuk-visually-hidden")
+  this.$chooseFilesButton.classList.remove("govuk-visually-hidden")
+
+  this.$chooseFilesButton.addEventListener('click', (e) => {
+    e.preventDefault() // prevent submitting form by default
+  })
+
+  // Display of files table (uploaded files only) is conditional on whether JS is enabled
+  let uploadTableJS = document.getElementById('upload-table-dropzone')
+  uploadTableJS.classList.remove("app-evidence-upload-hidden")
+  let uploadTableNonJS = document.getElementById('upload-table-fallback')
+  uploadTableNonJS.classList.add("app-evidence-upload-hidden")
```

Comment:
If js is not enabled, would this still be able to run?

Reply:
This only needs to run in JS as the default is to make the non js component visible and hide the dropzone component

Reply:
Oh, I see - I'd misread it. There are 2 tables that you are toggling between